src/dispatchers/browserTypeDispatcher.ts

```diff
@@ -37,4 +37,12 @@ export class BrowserTypeDispatcher extends Dispatcher<BrowserType, channels.Brow
     const browserContext = await this._object.launchPersistentContext(params.userDataDir, params);
     return { context: new BrowserContextDispatcher(this._scope, browserContext) };
   }
+
+  async cdpConnect(params: channels.BrowserTypeCdpConnectParams): Promise<channels.BrowserTypeCdpConnectResult> {
+    const browser = await this._object.cdpConnect(params.wsEndpoint, params, params.timeout);
+    return {
+      browser: new BrowserDispatcher(this._scope, browser),
+      defaultContext: new BrowserContextDispatcher(this._scope, browser._defaultContext!),
```

Comment:
How are you sure there is a default context? If I launched with Playwright, there might be none.